common/src/service/HttpClientHelper.cs

```diff
@@ -70,15 +74,15 @@ internal sealed class HttpClientHelper : IHttpClientHelper
             // We need two types of HttpClients, one with our default operation timeout, and one without. The one without will rely on
             // a cancellation token.
 
-            _httpClientWithDefaultTimeout = new HttpClient(_httpClientHandler)
+            _httpClientWithDefaultTimeout = new HttpClient(_httpClientHandler, false)
```

Comment:
False means don't dispose the handler. We'll do it ourselves because we share it with the 2 HttpClient instances.